### PR TITLE
fa: add government alias

### PR DESCRIPTION
**Pull request by Joery**
Add fa-government alias to the existing fa-university icon

### CODE REVIEW

**Comment by tagliala** [comment]
https://github.com/FortAwesome/Font-Awesome/pull/8222#issuecomment-167353251

> @Joery please add a filter, not an alias
so you will still need to alias `fa-government` to `fa-university` in your own application but when you search for `government` at http://fontawesome.io/icons/ you will find the icon you need

**Comment by Joery** [comment]
@tagliala
You have a example PR?

File: src/icons.yml [hunk 1]
```diff
@@ -3958,6 +3958,7 @@ icons:
     aliases:
       - institution
       - bank
+      - government
```
Comment (tagliala) [inline src/icons.yml]: just add a `filter:` line before this line

filter should be 4 space indented, like `aliases:`

**Comment by tagliala** [comment]
Sure.

See my above comment. Example PR: #7444

**Pull request closed without merging**